terraform/hcl/applicable/req_sw_terraform_aws_cloudfront_tls_only.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_cloudfront_distribution" "vulnerable_example" {
+  default_cache_behavior {
+    viewer_protocol_policy = "allow-all"
+
+    target_origin_id = "dummy"
+    allowed_methods  = ["dummy"]
+    cached_methods  = ["dummy"]
+  }
+
+  enabled             = true
+  origin {
+    origin_id = "dummy"
+    domain_name = "dummy"
+  }
+  restrictions {
+    geo_restriction {
+        restriction_type = "none"
+    }
+  }
+  viewer_certificate {
+  }
+}
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 22 2 vulnerable_example /terraform/hcl/applicable/req_sw_terraform_aws_cloudfront_tls_only.tf)

### 🛠️ Infrastructure as Code Vulnerability
	
| Severity | Finding |
| :---: | :---: |
| ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableLowSeverity.png)<br> | Deprecated TLS version was detected |

<details>
<summary> <b>Full description</b> </summary>
<br>

Resource `aws_cloudfront_distribution` should have `viewer_certificate.minimum_protocol_version = 'TLSv1.2*'` or `viewer_certificate.minimum_protocol_version = 'TLSv1.3*'`

Vulnerable example -
```
resource "aws_cloudfront_distribution" "vulnerable_example" {
  viewer_certificate {
    minimum_protocol_version  = "TLSv1"
  }
}
```

Secure example -
```
resource "aws_cloudfront_distribution" "secure_example" {
  viewer_certificate {
    minimum_protocol_version  = "TLSv1.2_2019"
  }
}

```

</details>


---
<div align="center">

[🐸 JFrog Frogbot](https://github.com/jfrog/frogbot#readme)

</div>